Context/SearchContextInterface.php

```diff
@@ -90,4 +90,11 @@ public function getFacetSortOrderProvider();
      * @return \Markup\NeedleBundle\Intercept\InterceptorInterface
      **/
     public function getInterceptor();
+
```

Comment:
It's needed by the purpose of this PR (creating an intermediate object)?

I don't disagree with its inclusion, it just seems like an irrelevant change to this PR.